src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.targets

```diff
@@ -95,6 +95,7 @@
     <HttpActivityPropagationSupport Condition="'$(HttpActivityPropagationSupport)' == ''">false</HttpActivityPropagationSupport>
     <InvariantGlobalization Condition="'$(InvariantGlobalization)' == ''">false</InvariantGlobalization>
     <StartupHookSupport Condition="'$(StartupHookSupport)' == ''">false</StartupHookSupport>
+    <UseNativeHttpHandler Condition=" $(AndroidHttpClientHandlerType.Contains ('System.Net.Http.SocketsHttpHandler')) And '$(UseNativeHttpHandler)' == '' ">false</UseNativeHttpHandler>
```

Comment:
@dellis1972 : this feels like something that should be called out in the commit message.  What's the rationale here?

Reply:
I think this is left over. But we got info from the runtime team that if a user defines `System.Net.Http.SocketsHttpHandler'` as the handler type they MUST use set the `UseNativeHttpHandler` msbuild property. Otherwise it doesn't work.

This is where the conversaton took place https://discord.com/channels/732297728826277939/732297837953679412/1029777522897993748

Reply:
Should we remove it ("this is left over") or keep it ("extract rationale from Discord")?